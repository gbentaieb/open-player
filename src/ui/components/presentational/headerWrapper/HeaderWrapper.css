.HeaderWrapper {
  position: absolute;
  display: flex;
  text-align: center;
  top: 0;
  padding: 5px 5px 5px 5px;
  height: 65px;
  width: calc(100% - 10px);
  z-index: 10;
}
