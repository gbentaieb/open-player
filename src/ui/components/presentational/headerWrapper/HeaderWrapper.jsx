import React, { Component } from 'react';
import PropTypes from 'prop-types';

import style from './HeaderWrapper.css';

class HeaderWrapper extends Component {
  static propTypes = {
    children: PropTypes.node,
  }

  static defaultProps = {
    children: <div />,
  }

  render() {
    return (
      <div className={style.HeaderWrapper}>
        {this.props.children}
      </div>
    );
  }
}

export default HeaderWrapper;
