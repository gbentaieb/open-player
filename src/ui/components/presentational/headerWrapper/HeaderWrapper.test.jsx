import React from 'react';
import HeaderWrapper from './HeaderWrapper';

const defaultProps = {
  children: null,
};

describe('HeaderWrapper snapshot', () => {
  test('HeaderWrapper > Snapshot', () => {
    const wrapper = shallow(<HeaderWrapper {...defaultProps} />);
    expect(wrapper).toMatchSnapshot();
  });
});
