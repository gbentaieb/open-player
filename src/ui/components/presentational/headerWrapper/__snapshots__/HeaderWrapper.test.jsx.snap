// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HeaderWrapper snapshot HeaderWrapper > Snapshot 1`] = `<div />`;
